SEED-platform/seed
# Org-level units preference

#### Any background context you want to provide?

This is in preparation to adding metric units and is a (hopefully) easy PR to use to shake out our review and merge process, code quality, test requirements etc.

#### What's this PR do?

Adds an enum setting to Organizations in the database, and adds a select in the organization settings to specify the preferred display and import units for users under that organization.

#### How should this be manually tested?

Apply migrations. Under `/app/#/accounts/n` for some org id, confirm the select is set to "United States customary units". Uncomment the metric setting (see 3a2a1af6) and try to save the preferred measurement as "Metric", coming back to the screen later to confirm it saves and reloads.

#### What are the relevant tickets?

n/a.

#### Screenshots (if appropriate)

<img width="614" alt="screen shot 2017-08-21 at 11 58 32" src="https://user-images.githubusercontent.com/6805/29533735-1c9d35f2-8668-11e7-81a6-0b9ba1a7193f.png">

#### Definition of Done:
